projects/extension/src/components/NetworkTab.tsx

```diff
@@ -109,58 +109,47 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 interface NetworkContentProps {
-  genesisHash: string
-  headerNumber: string
-  chainName: string
-  epoch: number
-  existentialDeposit: string
-  healthPeers: number
+  health: NetHealth;
+  apps: App[];
 }
 
 const NetworkContent = ({
-  genesisHash,
-  headerNumber,
-  chainName,
-  epoch,
-  existentialDeposit,
-  healthPeers
+  health,
+  apps
 }: NetworkContentProps) => {
   const classes = useStyles();
-
-  return headerNumber ? (
-    <div className={classes.info}>
-      <p>{emojis.tick} Connected to <b>{chainName}</b></p>
-      <p>{emojis.seedling} Syncing will start at block #{headerNumber}</p>
-      <p>{emojis.chequeredFlag} Genesis hash is {genesisHash}</p>
-      <p>{emojis.clock} Epoch duration is {epoch} blocks</p>
-      <p>{emojis.info} Existential deposit is {existentialDeposit}</p>
-      <p>{emojis.star} Communicating with {healthPeers} peer{healthPeers === 1 ? '' : 's'}</p>
-    </div>) : (
+  const peers = health && health.peers;
+  const status = health && health.status;
+  const shouldHavePeers = health && health.shouldHavePeers;
+  const isSyncing = health && health.isSyncing;
+  return (
     <div className={classes.info}>
-      <p>{emojis.chain} Syncing <b>{chainName}</b></p>
+      <p>{emojis.chain} Chain is {status} and should {!shouldHavePeers && 'not'} have peers.</p>
+      <p>{emojis.star} Communicating with {peers} peer{peers === 1 ? '' : 's'}.</p>
+      <p>{emojis.info} GrandPa sync is {isSyncing && 'not'} finished.</p>
```

Comment:
I don't think we should mention Grandpa warp syncing. Users don't care about Grandpa warp syncing, they just want to know whether everything is working correctly or not.

Reply:
I will remove Grandpa wording then. I think user should be aware if everything is synced or not. What do you think @tomaka ?

Reply:
Yeah they need to be aware whether it's synced, but in my opinion showing either :heavy_check_mark: or a red X works as well. What I'm not a fan of is talking about "Grandpa warp sync" in particular.